### ISSUE TITLE
World Scale

### ISSUE THREAD

**Issue by cercata** [xLAva/JediAcademyLinux#19]
Hey people, how do you feel world scale ?

I feel it too big, and with the same setup in Jedi Outcast, it feels OK to me.

I'm so glad I didn't play these games back in their time, so now I have 2 brand new VR games !!!!

**Comment by xLAva** [comment]
I think it is a bit too big too. I experimented in Jedi Outcast with a new scale. I'm going to use the same scale for Jedi Academy soon.

**Comment by cercata** [comment]
Cool, It wasn't my perception fooling me in the virtual world !!!!

Amazing work dude, it's so fluid inside those games !!!!!

**Comment by xLAva** [comment]
I'm glad there are people out there playing it.
If you encounter any bugs, just report them here. If I have time I will try to fix them.

**Comment by cercata** [comment]
OK, i'll do.

BTW, do you use the IPD from the API for scaling ?

**Comment by xLAva** [comment]
The IPD is not the problem (and yes it is used). The problem is, that the unit inside the Jedi games is not meter or feet but something different. I thought I found somewhere that it is in feet but scaled by 2. This is the current setting of Jedi Academy, but as you said yourself it feels to big. I also tried without the scale by 2, but that feels way too small, so I just experimented with different scaling values between both and I think I found a good value now in Jedi Outcast.
